source/_posts/2024-12-04-release-202412.markdown

```diff
@@ -0,0 +1,644 @@
+
+{% details "Brother Printer" %}
+
+The unit of measurement for pages changed from `p` to `pages`.
+
+This could potentially break use cases if it was used in templates where a state
+with a unit is used, such as `states('sensor.pages', with_unit=True)`.
+
+([@bieniu] - [#131275]) ([documentation](/integrations/brother))
+
+[@bieniu]: https://github.com/bieniu
+[#131275]: https://github.com/home-assistant/core/pull/131275
+
+{% enddetails %}
+
+{% details "Ecovacs" %}
+
+Ecovacs devices not recognized by the library will no longer use the fallback
+vacuum. Instead, the device will not be added to Home Assistant,
+and a warning log will created.
+
+([@edenhaus] - [#131525]) ([documentation](/integrations/ecovacs))
+
+[@edenhaus]: https://github.com/edenhaus
+[#131525]: https://github.com/home-assistant/core/pull/131525
+
+{% enddetails %}
+
+{% details "Glances" %}
+
+The previously deprecated Glances v2 API has been removed. Upgrade to v3 or
+higher to continue using the integration.
+
+([@gjohansson-ST] - [#131427]) ([documentation](/integrations/glances))
+
+[@gjohansson-ST]: https://github.com/gjohansson-ST
+[#131427]: https://github.com/home-assistant/core/pull/131427
+
+{% enddetails %}
+
+{% details "History Stats" %}
+
+After this change, `history_stats` in `time` or `ratio` mode will only count
+definitively known time-based on recorded data.
+
+Prior to this change, if the start of the time window was earlier than the
+oldest datapoint in the recorder, `history_stats` would count time using the
+assumption that an entity was in the first state it found since the beginning
+of time. This could lead to wild overcalculations when a switch was turned on
+after a long period of inactivity, and `history_stats` would assume it has
+_always_ been on and calculate a huge value based on that.
+
+([@karwosts] - [#126271]) ([documentation](/integrations/history_stats))
+
+[@karwosts]: https://github.com/karwosts
+[#126271]: https://github.com/home-assistant/core/pull/126271
+
+{% enddetails %}
+
+{% details "Jewish Calendar" %}
+
+To support Rosh Chodesh this creates a breaking change as the holiday sensor
+will return "Chanukah, Rosh Chodesh" when both of those days fall on the same
+date. If you have an automation expecting the "Chanukah" value, you must update
+it.
+
+([@tsvi] - [#130456]) ([documentation](/integrations/jewish_calendar))
+
+[@tsvi]: https://github.com/tsvi
+[#130456]: https://github.com/home-assistant/core/pull/130456
+
+{% enddetails %}
+
+{% details "QNAP" %}
+
+The following QNAP sensors will no longer have extra state attributes:
+
+- `memory_free`
+- `memory_percent_used`
+- `memory_size`
+- `memory_used`
+- `network_err`
+- `network_link_status`
+- `network_max_speed`
+- `network_rx`
+- `network_tx`
+- `status`
+- `system_temp`
+- `uptime`
+- `volume_percentage_used`
+- `volume_size_free`
+- `volume_size_total`
+- `volume_size_used`
+
+These extra state attributes were deprecated in Home Assistant 2024.6 and have
```

Comment:
Are the listed items both sensors and extra attributes?

Reply:
Extra attributes as it seems.

Reply:
Above the list it says these sensors.

Reply:
Fixed